WFP-VAM/prism-app
# Compare charts over location/time periods; COUNTRY=cambodia

This fixes issue #930 and #931 

How to test the feature:

- go to the charts tab
- click the "compare locations" switch
- select locations 1 & 2
- select time period and chart layers
- see charts show up

## TODO
- [x] Provide 2 CSV files when comparing 
- [x] Limit number of selectable layers to 2 (is this really needed?)
- [x] Adjust styling
- [x] Add time period comparison


Screenshot/video of feature:
<img width="1727" alt="Screenshot 2023-10-23 at 5 18 38 PM" src="https://github.com/WFP-VAM/prism-app/assets/16843267/9f7a9dd9-16d7-42f7-af7c-c913986e30b2">
